tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.csproj

```diff
@@ -64,7 +64,9 @@
       </None>
       <None Include="$(IntermediateOutputPath)/$(AssemblyName).dll" Pack="true" PackagePath="tasks/" Visible="false" />
       <!-- Use the compilers from this source build. -->
-      <None Include="../IceRpc.ProtocGen/bin/$(Configuration)/net8.0/*" Exclude="../IceRpc.ProtocGen/bin/$(Configuration)/net8.0/*.exe">
+      <None
```

Comment:
This was an issue before because `net8.0` was hardcoded here. We cannot use `TargetFramework` because for this project `TargetFramework` must remain set to `netstandard2.0`.

Reply:
Here, we import IceRpc.Version.props at the top of the file, so it's clear where it comes from.